### PR TITLE
Feature/small misc optimizations

### PR DESCRIPTION
**Pull request by jsteemann**
- separate key & value in agency log message, as we do it in other places
- fix broken include guard
- centralize strings for HTTP headers
- don't copy HTTP headers all the time

### CODE REVIEW

File: arangod/VocBase/Methods/Transactions.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #ifndef ARANGOD_VOCBASE_METHODS_TRANSACTIONS_HANDLER_H
-#define ARANGOD_VOCBASW_METHODS_TRANSACTIONS_HANDLER_H 1
+#define ARANGOD_VOCBASE_METHODS_TRANSACTIONS_HANDLER_H 1
```
Comment (kvahed) [inline arangod/VocBase/Methods/Transactions.h]: LOL

File: lib/Rest/GeneralResponse.h [hunk 2]
```diff
@@ -158,7 +158,7 @@ class GeneralResponse {
   bool generateBody() const { return _generateBody; };  // used for head
```
Comment (kvahed) [inline lib/Rest/GeneralResponse.h]: OK then, remove the ';' also at end of  line 107

Comment (jsteemann) [reply lib/Rest/GeneralResponse.h]: yes, will do!

Comment (jsteemann) [reply lib/Rest/GeneralResponse.h]: ok, done.

**Pull request merged**